rtptransceiver.go

```diff
@@ -80,6 +81,33 @@ func (t *RTPTransceiver) getCodecs() []RTPCodecParameters {
 	return filteredCodecs
 }
 
+// SetHeaderExtensions stores header extension from SDP
+func (t *RTPTransceiver) SetHeaderExtensions(headerExtensions []RTPHeaderExtensionParameter) error {
+	t.mu.Lock()
+	defer t.mu.Unlock()
+
+	t.headerExtensions = headerExtensions
+	return nil
+}
+
+func (t *RTPTransceiver) hasHeaderExtension(uri string) bool {
+	t.mu.RLock()
+	defer t.mu.RUnlock()
+
+	// if nothing specific set, accept all
+	if len(t.headerExtensions) == 0 {
+		return true
+	}
```

Comment:
Had to add this accept all case to fix a test. But, I think this is fine to keep the old behaviour.